### PR TITLE
Better error handling for getTeams function

### PR DESCRIPTION
**Pull request by hubertdeng123**
There needs to be better error-handling here, so adding that in

### CODE REVIEW

File: src/utils/getTeams.test.ts [hunk 1]
```diff
@@ -28,4 +30,28 @@ describe('getTeams', function () {
       'team-enterprise',
     ]);
   });
+
+  it('should return empty array if org is codecov', () => {
```
Comment (azaslavsky) [inline src/utils/getTeams.test.ts:34]: Why does this and the test below have the same behavior? Is this just to verify that codecov is itself an undefined team?

File: src/utils/getTeams.ts [hunk 2]
```diff
@@ -6,11 +8,23 @@ export function getTeams(
   org: string
 ): string[] {
   const orgObj = GH_ORGS.get(org);
+  // TODO: include codecov
+  if (org === 'codecov') {
+    return [];
```
Comment (azaslavsky) [inline src/utils/getTeams.ts:13]: Seems like this line is not hit by any tests?

**Review by azaslavsky — APPROVED** [review]

File: src/utils/getTeams.ts (hunk 2 above)
Comment (hubertdeng123) [reply src/utils/getTeams.ts:13]: my test case appears to be wrong, fixing it now

File: src/utils/getTeams.test.ts (hunk 1 above)
Comment (hubertdeng123) [reply src/utils/getTeams.test.ts:34]: Yeah, if any event comes from the codecov org let's just return an empty array for teams temporarily until we roll this out to them

File: src/utils/getTeams.ts (hunk 2 above)
Comment (hubertdeng123) [reply src/utils/getTeams.ts:13]: fixed now.

**Pull request merged**